TetrisDotnet/Code/UI/Layouts/GameLayout.cs

```diff
@@ -0,0 +1,7 @@
+namespace TetrisDotnet.Code.UI.Layouts
+{
+	public class GameLayout
```

Comment:
Is this the UI layout?

Reply:
Yeah, Haven't gotten around to actually using this though. I wanted the GameLayout to be inside the GameScene, then all UI code could go into GameLayout instead, to decouple a bit better.